x-pack/plugins/uptime/public/pages/monitor_management/use_monitor_management_breadcrumbs.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+import { useBreadcrumbs } from '../../hooks/use_breadcrumbs';
+import { MONITOR_MANAGEMENT_ROUTE } from '../../../common/constants';
+import { PLUGIN } from '../../../common/constants/plugin';
+
+export const useMonitorManagementBreadcrumbs = ({
```

Comment:
would be nice to add few unit tests for this hook.

Reply:
I was starting to, and it took a bit more boilerplate than I wanted. @shahzad31 do you think it would be okay to test this with @elastic/synthetics

Reply:
i am fine, if it has e2e tests ++